nifi-nar-bundles/nifi-standard-bundle/nifi-standard-parameter-providers/src/main/resources/docs/org.apache.nifi.parameter.DatabaseParameterProvider/additionalDetails.html

```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/html">
+<!--
+      Licensed to the Apache Software Foundation (ASF) under one or more
+      contributor license agreements.  See the NOTICE file distributed with
+      this work for additional information regarding copyright ownership.
+      The ASF licenses this file to You under the Apache License, Version 2.0
+      (the "License"); you may not use this file except in compliance with
+      the License.  You may obtain a copy of the License at
+          http://www.apache.org/licenses/LICENSE-2.0
+      Unless required by applicable law or agreed to in writing, software
+      distributed under the License is distributed on an "AS IS" BASIS,
+      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+      See the License for the specific language governing permissions and
+      limitations under the License.
+    -->
+
+<head>
+    <meta charset="utf-8"/>
+    <title>DatabaseParameterProvider</title>
+    <link rel="stylesheet" href="../../../../../css/component-usage.css" type="text/css"/>
+</head>
+<body>
+
+<h1>Providing Parameters from a Database</h1>
+
+<p>
+    The DatabaseParameterProvider at its core maps database rows to Parameters, specified by a
+    Parameter Name Column and Parameter Value Column.  The Parameter Group name must also be accounted for, and may
+    be specified in different ways using the Parameter Grouping Strategy.
+</p>
+
+<p>
+    The default configuration uses a fully column-based approach, with the Parameter Group Name
+    also specified by columns in the same table.  An example of a table using this configuration would be:
+</p>
+<table>
+    <thead>
+        <tr>
+            <th colspan="4" style="text-align: center">PARAMETER_CONTEXTS</th>
+        </tr>
+        <tr>
+            <th>PARAMETER_NAME</th><th>PARAMETER_VALUE</th><th>PARAMETER_GROUP</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>param.foo</td><td>value-foo</td><td>group_1</td>
+        </tr>
+        <tr>
+            <td>param.bar</td><td>value-bar</td><td>group_1</td>
+        </tr>
+        <tr>
+            <td>param.one</td><td>value-one</td><td>group_2</td>
+        </tr>
+        <tr>
+            <td>param.two</td><td>value-two</td><td>group_2</td>
+        </tr>
+    </tbody>
+    <caption>Table 1: Database table example with Grouping Strategy = Column</caption>
+</table>
+
+<p>
+    In order to use the data from this table, set the following Properties:
+</p>
+
+<ul>
+    <li><b>Parameter Grouping Strategy</b> - Column</li>
+    <li><b>Table Name</b> - PARAMETER_CONTEXTS</li>
+    <li><b>Parameter Name Column</b> - PARAMETER_NAME</li>
+    <li><b>Parameter Value Column</b> - PARAMETER_VALUE</li>
+    <li><b>Parameter Group Name Column</b> - PARAMETER_GROUP</li>
+</ul>
+
+<p>
+    Note: in some databases, the words 'PARAMETER', 'PARAMETERS', 'GROUP', and even 'VALUE' are reserved words.
```

Comment:
Maybe add to this to check the database docs and/or quote the words per the DB doc